perfkitbenchmarker/linux_benchmarks/cluster_boot_benchmark.py

```diff
@@ -83,7 +83,26 @@ def Run(benchmark_spec):
   samples = vm_util.RunThreaded(_GetTimeToBoot, params)
   logging.info(samples)
   assert len(samples) == len(vms)
-  return samples
+  total_boot = benchmark_spec.vm_boot_endtime - benchmark_spec.vm_boot_starttime
```

Comment:
Yes, it's not really a pure boot time measurement or a time-to-ssh measurement.  I should have been clearer in the original pull request.  

The problem is that the benchmark gives time-to-ssh for each individual machine.  I've been considering the maximum of those numbers as the overall cluster time-to-ssh, but that's not quite right.  This number is also not really a cluster-time-to-ready either, for the reasons you mention.  

There are some options:
* Don't try to output a number representing how long the cluster took to get ready, and just use the maximum time-to-ssh.
*  Output this number, but rename it something else ('Cluster Ready Time', 'Cluster Start Time', 'Cluster Start and Configure Time')
* Refactor the code so that it's possible to measure of how long it takes for all hosts in the cluster to be ssh-able.   

Of those options, I prefer option 2.